client.go

```diff
@@ -599,6 +602,20 @@ func (c *client) DesiredLRPSchedulingInfos(logger lager.Logger, filter models.De
 	return response.DesiredLrpSchedulingInfos, response.Error.ToError()
 }
 
+func (c *client) DesiredLRPRoutingInfos(logger lager.Logger, filter models.DesiredLRPFilter) ([]*models.DesiredLRP, error) {
+	request := models.DesiredLRPsRequest{
+		Domain:       filter.Domain,
```

Comment:
Perhaps 'Domain' is not required here as we request filtering based on guid only in route-emitter